---
title: Highly Accurate Estimation of Cell Type Abundance in Bulk Tissues Based on
  Single‚ÄêCell Reference and Domain Adaptive Matching
authors:
- Xinyang Guo
- admin
- Fen Ju
- Chenguang Zhao
- Liang Yu
date: '2024-01-01'
publishDate: '2025-02-05T11:53:58.901347Z'
publication_types:
- article-journal
publication: '*Advanced Science*'

# Mannual add
url_pdf: https://advanced.onlinelibrary.wiley.com/doi/epdf/10.1002/advs.202306329
url_code: https://github.com/LiangYu-Xidian/SCROAM

abstract: Accurately identifies the cellular composition of complex tissues, which is critical for understanding disease pathogenesis, early diagnosis, and prevention. However, current methods for deconvoluting bulk RNA sequencing (RNA-seq) typically rely on matched single-cell RNA sequencing (scRNA-seq) as a reference, which can be limiting due to differences in sequencing distribution and the potential for invalid information from single-cell references. Hence, a novel computational method named SCROAM is introduced to address these challenges. SCROAM transforms scRNA-seq and bulk RNA-seq into a shared feature space, effectively eliminating distributional differences in the latent space. Subsequently, cell-type-specific expression matrices are generated from the scRNA-seq data, facilitating the precise identification of cell types within bulk tissues. The performance of SCROAM is assessed through benchmarking against simulated and real datasets, demonstrating its accuracy and robustness. To further validate SCROAM's performance, single-cell and bulk RNA-seq experiments are conducted on mouse spinal cord tissue, with SCROAM applied to identify cell types in bulk tissue. Results indicate that SCROAM is a highly effective tool for identifying similar cell types. An integrated analysis of liver cancer and primary glioblastoma is then performed. Overall, this research offers a novel perspective for delivering precise insights into disease pathogenesis and potential therapeutic strategies.
---
